src/test/java/org/broadinstitute/hellbender/tools/PrintReadsIntegrationTest.java

```diff
@@ -215,4 +217,19 @@ public void testUnmappedReadInclusion( final File input, final String reference,
             }
         }
     }
+    @Test(expectedExceptions = UserException.MissingReference.class)
+    public void testNonExistentReference() throws Exception {
```

Comment:
Might want to put this in something like `GATKToolUnitTest` rather than embedded in the tests for `PrintReads`